man/docker.1.md

```diff
@@ -19,42 +19,42 @@ To see the man page for a command run **man docker <command>**.
 
 # OPTIONS
 **--help**
-  Print usage statement
+  Print usage statement.
 
-**--config**=""
-  Specifies the location of the Docker client configuration files. The default is '~/.docker'.
+**--config** ""
+  Specifies the location of the Docker client configuration files. The default is `~/.docker`.
```

Comment:
By man convention, filenames are rendered in italic, i.e. it should be `*~/.docker*`.